homeassistant/components/here_travel_time/sensor.py

```diff
@@ -277,175 +264,32 @@ def extra_state_attributes(
         self,
     ) -> dict[str, None | float | str | bool] | None:
         """Return the state attributes."""
-        if self._here_data.base_time is None:
-            return None
-
-        res = self._attrs
-        if self._here_data.attribution is not None:
-            res[ATTR_ATTRIBUTION] = self._here_data.attribution
-        res[ATTR_DURATION] = self._here_data.base_time / 60
-        res[ATTR_DISTANCE] = self._here_data.distance
-        res[ATTR_ROUTE] = self._here_data.route
-        res[ATTR_DURATION_IN_TRAFFIC] = self._here_data.traffic_time / 60
-        res[ATTR_ORIGIN] = self._here_data.origin
-        res[ATTR_DESTINATION] = self._here_data.destination
-        res[ATTR_ORIGIN_NAME] = self._here_data.origin_name
-        res[ATTR_DESTINATION_NAME] = self._here_data.destination_name
-        return res
+        if self.coordinator.data is not None:
+            res = {
+                ATTR_UNIT_SYSTEM: self.coordinator.config.units,
+                ATTR_MODE: self.coordinator.config.travel_mode,
+                ATTR_TRAFFIC_MODE: self._traffic_mode,
+                **self.coordinator.data,
+            }
+            if res[ATTR_ATTRIBUTION] is None:
+                res.pop(ATTR_ATTRIBUTION)
```

Comment:
Attribution should be in attribution and not in state_attributes
https://github.com/home-assistant/core/blob/0aaf0b616081f534055009e8592ddddf17f5aa91/homeassistant/helpers/entity.py#L438

Reply:
done